File: pinyougou-manager-web/src/main/webapp/js/service/brandservice.js
/*
		* 服务层
		* */
		app.service('brandService',function ($http) {
			this.findAll=function () {
				return $http.get('../brand/findAll.do')
			}

			this.search=function (pageNum, pageSize,searchEntity) {
				return $http.post('../brand/search.do?pageNum='+pageNum+'&pageSize='+pageSize,searchEntity)
			}

			this.save=function (entity) {
				return $http.post('../brand/save.do',entity)
			}

			this.updateBrand=function (entity) {
				return $http.post('../brand/updateBrand.do',entity)
			}

			this.findOne=function (id) {
				return $http.post('../brand/findOne.do?id='+id)
			}
			this.dele=function (selectIds) {
				return $http.post("../brand/delect.do?ids="+selectIds)
			}

			//select2 信息数据
			this.selectOptionList=function () {
				return $http.post("../brand/selectOptionList.do")
			}


            //获取用户名
            this.loginName=function () {
                return $http.post("../brand/name.do")
            }
		});

File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import entity.PageResult;
import com.pinyougou.pojo.TbBrand;

import java.util.List;
import java.util.Map;

public interface BrandService {

   /*查询品牌列表*/
   public List<TbBrand> findAll();

   /*查找分页列表*/
   public PageResult findPage(int pageNum,int pageSize);

   /*修改品牌信息*/
   void updateBrand(TbBrand tbBrand);
   /*id查找  回显数据*/
   TbBrand findOne(long id);

   /*添加品牌*/
   void save(TbBrand tbBrand);

   /*删除数据*/
   void delect(long[] id);

   /*条件查询*/
   PageResult findPage(TbBrand tbBrand,int pageNum,int pageSize);

   /*select2 下拉列表的数据*/
   List<Map> selectOptionList();
}
